lib/libvarnish/vsb.c

```diff
@@ -529,7 +529,7 @@ VSB_quote_pfx(struct vsb *s, const char *pfx, const void *v, int len, int how)
 {
 	const uint8_t *p = v;
 	const uint8_t *q;
-	int quote = 0;
+	int quote = 0, dq = 0;
```

Comment:
Flexelint:

> Info 838: Previously assigned value to variable 'dq' has not been used

Just remove the initialization please.

Reply:
fixed